test/spec/modules/yieldlabBidAdapter_spec.js

```diff
@@ -567,15 +576,24 @@ describe('yieldlabBidAdapter', () => {
       expect(result[0].native.image.url).to.equal('https://localhost:8080/yl-logo100x100.jpg');
       expect(result[0].native.image.width).to.equal(100);
       expect(result[0].native.image.height).to.equal(100);
+      expect(result[0].native.icon.url).to.equal('https://localhost:8080/assets/favicon/favicon-16x16.png');
+      expect(result[0].native.icon.width).to.equal(16);
```

Comment:
Comparing with the [spec on page 22](https://www.iab.com/wp-content/uploads/2018/03/OpenRTB-Native-Ads-Specification-Final-1.2.pdf) I think this needs to be changed to `w` and `h` and also for the probably wrong expectation above.